### PR TITLE
feat: Complete UI e2e tests

### PR DESCRIPTION
**Pull request by ajborley**
Completes the e2e tests for the UI, running against the 
mock vertx websocket server.

Contributes to: matthew-chirgwin/kafka-java-vertx-starter#60

Signed-off-by: Andrew Borley <borley@uk.ibm.com>

### CODE REVIEW

File: ui/jest-playwright.config.js [hunk 1]
```diff
@@ -13,5 +13,5 @@ module.exports = {
   launchOptions: {
     headless: true,
   },
-  browsers: ['firefox', 'webkit', 'chromium'],
+  browsers: ['firefox', 'chromium'],
```
Comment (ajborley) [inline ui/jest-playwright.config.js:16]: Matthew and I found an issue when running the UI in Webkit/Safari - it appears that the creation of 2 websockets in close order causes the first of the two to never complete its creation (the onOpen event is never called), meaning that the producer side of the UI never becomes available. Occasionally we did see both websockets happily created, so there is some kind of timing window problem that is only seen on Webkit/Safari.

For this PR, this meant that we saw random failures in the e2e tests, hence temporarily removing webkit from the list of browsers here.

File: ui/src/DevUtils/MockVertx/MockVertxServer.js [hunk 2]
```diff
@@ -212,7 +232,7 @@ const defaultConfig = {
   // consume records endpoint
   consumeEndpoint: CONSUME_ENDPOINT,
   // enable/disable extra logging
-  enableLog: false,
+  enableLog: true,
```
Comment (matthew-chirgwin) [inline ui/src/DevUtils/MockVertx/MockVertxServer.js:235]: Being hyper picky, should this really be enabled by default? I know we enabled it to debug the webkit websocket issue - was it useful to leave on?

Comment (ajborley) [reply ui/src/DevUtils/MockVertx/MockVertxServer.js]: I missed that change - have reverted it.

**Comment by katheris** [comment]
@matthew-chirgwin this has passed, if you're happy we can merge it

**Comment by matthew-chirgwin** [comment]
@ajborley and my working theory is that there may be a dependancy issue @katheris - I will re run the tests a few more times to see if they reliably run (they were not locally or via github yesterday). I will merge if we get some consistent results

**Comment by matthew-chirgwin** [comment]
This is running consistantly now, so will approve and merge - thank you @ajborley !

**Pull request merged**